docs/layouts/shortcodes/generated/catalog_configuration.html

```diff
@@ -116,6 +122,12 @@
             <td>String</td>
             <td>Metastore of paimon catalog, supports filesystem, hive and jdbc.</td>
         </tr>
+        <tr>
+            <td><h5>resolving-file-io.enabled</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>Whether to enable resolving fileio, when this option is enabled, in conjunction with the table's property data-file.external-paths, Paimon can read and write to external storage paths, such as OSS or S3. In order to access these external paths correctly, you also need to configure the corresponding access key and secret key.</td>
```

Comment:
fileio -> file io same with line 87?

Reply:
it is ok in description, we can just be careful for option key, it is API part.